test/common/gitops_utils.go

```diff
@@ -72,15 +103,21 @@ func GitOpsUserSetup(
 	}
 
 	By("Cleaning up any existing subscription-admin user config")
-	GitOpsCleanup(namespace, ocpUser)
+	GitOpsCleanup(ocpUser)
+
+	for _, ns := range gitopsTestNamespaces {
+		CleanupHubNamespace(ns)()
```

Comment:
Wait a min. What's up with that extra set of parentheses?

Reply:
Oh wild. I was wondering about this when I saw the `Eventually`. So it's returning a function so that it uses the Gomega that's currently in action? Why not just pass the Gomega object in instead of returning an entire function (assuming that's the reason)?

Reply:
It's a function that returns a function. The most common use of the function is:
```go
AfterAll(CleanupHubNamespace("foobar"))
```
because `AfterAll` takes a function.

It could be written to be less weird here, but then the more usual case would look like:
```go
AfterAll(func() {
    CleanupHubNamespace("foobar")
})

Reply:
I do like how much cleaner it is, but I think in the end it's confusing since it won't look like the other AfterAlls() and if I want to use `CleanupHubNamespace()` alongside other functions, I'd need to use the funky syntax.

Reply:
Good point. Updated 👍